packages/core/src/manifest.ts

```diff
@@ -38,27 +40,82 @@ function defaultManifest(): ManifestData {
   };
 }
 
-const manifestDir = '.openzeppelin';
+const MANIFEST_DEFAULT_DIR = '.openzeppelin';
+const MANIFEST_TEMP_DIR = 'openzeppelin';
```

Comment:
Something more specific might be safer:

```suggestion
const MANIFEST_TEMP_DIR = 'openzeppelin-upgrades';
```